src/Mvc/Mvc.NewtonsoftJson/test/NewtonsoftJsonInputFormatterTest.cs

```diff
@@ -14,15 +14,16 @@
 using Microsoft.Extensions.ObjectPool;
 using Moq;
 using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
 using Newtonsoft.Json.Serialization;
 using Xunit;
 
 namespace Microsoft.AspNetCore.Mvc.Formatters
 {
     public class NewtonsoftJsonInputFormatterTest : JsonInputFormatterTestBase
     {
-        private static readonly ObjectPoolProvider _objectPoolProvider = new DefaultObjectPoolProvider();
-        private static readonly JsonSerializerSettings _serializerSettings = new JsonSerializerSettings();
+        private readonly ObjectPoolProvider _objectPoolProvider = new DefaultObjectPoolProvider();
+        private readonly JsonSerializerSettings _serializerSettings = new JsonSerializerSettings();
```

Comment:
Why did this need to change?

Reply:
The newly added test mutates the serializerSettings to add a converter (which has the problematic behavior)